packages/sources/bank-frick/src/config/index.ts

```diff
@@ -0,0 +1,80 @@
+import { Logger, Requester, util } from '@chainlink/ea-bootstrap'
+import { Config as BootstrapConfig } from '@chainlink/ea-bootstrap'
+import { SigningAlgorithm } from '../types'
+
+export const NAME = 'POUND_TOKEN'
+export const DEFAULT_BASE_URL = 'https://olbsandbox.bankfrick.li/webapi/v2'
+export const DEFAULT_ENDPOINT = 'accounts'
+export const DEFAULT_PAGESIZE = 500
+export const PAGE_SIZE_MAX = 500
+export const PAGE_SIZE_MIN = 1
+export const DEFAULT_SIGNING_ALGORITHM: SigningAlgorithm = 'rsa-sha512'
+
+export type Config = BootstrapConfig & {
+  pageSize: number
+  privateKey: string
+  password: string
+  signingAlgorithm: SigningAlgorithm
+  token: string //Set as a global variable on the first run.
+  allowInsecure?: boolean //Sandbox's cert setup is difficult, so allow skipping verification in dev only.
+}
+
+//Global variable to keep the token. Token is provisioned when the accounts endpoint is hit.
+let token: string
+export const setToken = (newToken: string) => (token = newToken)
+
+export const makeConfig = (prefix?: string): Config => {
+  const baseConfig = Requester.getDefaultConfig(prefix)
+  const pageSizeString = util.getEnv('PAGE_SIZE')
```

Comment:
What's the motivation to set this as an env var instead of as an input parameter?

Reply:
I think it's fine for now. If we need to make changes it's easier (from a compatibility standpoint) to move env vars to input params than the other way around.